third_party/react-dates/index.js

```diff
@@ -1,2 +1,18 @@
 import 'react-dates/initialize';
 import 'react-dates';
+
+// A very basic polyfill for Array.from,
+// to avoid needing to use a full 3p implementation
+if (!Array.from) {
+  Array.from = function (arrayLike) {
+    if (!arrayLike) {
+      return [];
+    }
+
+    if (!arrayLike.length) {
+      return [];
+    }
+
+    return [].concat(arrayLike);
```

Comment:
Might be better to do a for loop? Concat will only spread arrays, not array-likes.